lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -193,7 +203,7 @@ getBlockHeader = label "getBlockHeader" $ do
         version <- getWord16be
         contentSize <- getWord32be
         slotEpoch <- fromIntegral <$> getWord32be
-        slotId <- toEnum . fromEnum <$> getWord32be
+        slotNo <- SlotNo <$> getWord32be
```

Comment:
We can also remove the `fromIntegral` at line 600 `SlotsPerEpoch` since it is changed too